### PR TITLE
[UI Services to Extensions] Inject into Extension Modules / improve tests

### PR DESCRIPTION
**Pull request by igoroctaviano**

### CODE REVIEW

File: platform/viewer/src/appExtensions/MeasurementsPanel/index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default {
   preRegistration({ servicesManager, configuration = {} }) {
     init({ servicesManager, configuration });
   },
-  getPanelModule() {
+  getPanelModule({ servicesManager }) {
```
Comment (dannyrb) [inline platform/viewer/src/appExtensions/MeasurementsPanel/index.js]: Should we pass this in as `servicesManager` or `ServicesManager`?

File: platform/core/src/services/ServicesManager.test.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      const fakeServices = [
+        { name: 'UINotificationTestService', hide: jest.fn() },
+        { name: 'UIModalTestService', hide: jest.fn() },
+      ];
+
+      servicesManager.registerServices(fakeServices);
+
+      expect(servicesManager.registerService.mock.calls.length).toBe(2);
+    });
+  });
```
Comment (dannyrb) [inline platform/core/src/services/ServicesManager.test.js]: Solid test :+1:
This is in line with the `registerExtensions` method for the ExtensionManager

File: platform/core/src/services/ServicesManager.test.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import ServicesManager from './ServicesManager.js';
+import log from '../log.js';
+
+jest.mock('./../log.js');
+
+describe('ServicesManager.js', () => {
```
Comment (dannyrb) [inline platform/core/src/services/ServicesManager.test.js]: Solid test suite! Great job 🥇

File: platform/viewer/src/appExtensions/MeasurementsPanel/index.js (hunk 1 above)
Comment (igoroctaviano) [reply platform/viewer/src/appExtensions/MeasurementsPanel/index.js]: I'm using camel here because it's an instance of ServicesManager

**Pull request merged**